webpack.config.js

```diff
@@ -20,6 +20,10 @@ const AssetsPlugin = require( 'assets-webpack-plugin' );
 const cacheIdentifier = require( './server/bundler/babel/babel-loader-cache-identifier' );
 const config = require( './server/config' );
 const UseMinifiedFiles = require( './server/bundler/webpack-plugins/use-minified-files' );
+const babelConfig = JSON.parse( fs.readFileSync( './.babelrc', { encoding: 'utf8' } ) );
+
+babelConfig.presets[ 0 ][ 1 ].modules = false;
+babelConfig.plugins = _.without( babelConfig.plugins, 'add-module-exports' );
```

Comment:
Is it introduced to keep the existing setup for server files and tests? 🤔

Reply:
Yep. And also the bin files that generate component statistics etc. More than a couple pieces rely on the current babelrc file, so I thought this was the cleanest for now.

Reply:
note that i am reverting this before merging. its just for people to have an easy time verifying w/ webpack that all of the imports/exports are configured properly.

Reply:
Yes, it all makes sense 👍